Doc/library/sqlite3.rst

```diff
@@ -1041,19 +1041,34 @@ Connection objects
          (2, 'broccoli pie', 'broccoli cheese onions flour')
          (3, 'pumpkin pie', 'pumpkin sugar flour butter')
 
-   .. method:: load_extension(path, /)
+   .. method:: load_extension(path, /, *, entrypoint=None)
 
-      Load an SQLite extension from a shared library located at *path*.
+      Load an SQLite extension from a shared library.
       Enable extension loading with :meth:`enable_load_extension` before
       calling this method.
 
+      :param str path:
+
+         The path to the SQLite extension.
+
+      :param entrypoint:
+
+         Optional entry point name.
+         If ``None`` (default),
+         SQLite will try to come up with an entry point name of its own.
```

Comment:
Wait, what does "try to come up with an entry point name of its own" mean? Does it randomly generate an entry point name? Is there a chance it will fail to come up with an entry point name of its own?

Reply:
Some context: [SQLite has its own entrypoint name resolution algorithm](https://www.sqlite.org/loadext.html#:~:text=Both%20methods%20for%20loading%20an%20extension) that depends on the extension's path. 

That paragraph describes it in detail, but as an example, if the extension path is `"/usr/lib/libmathfunc-4.8.so"`, it'll first try the default `sqlite3_extension_init ` entrypoint, and if that does't work, it'll apply the resolution algorithm to `sqlite3_mathfunc_init` and try that. 

Maybe we can include a link in these docs to that section in the SQLite docs?

Reply:
Yes, we can add a link to that section. I did not want to C&P the SQLite docs, since they IMO add to much detail. I'll try to come up with a better suggestion. I agree that the current wording is far from optimal.

Reply:
How about 89b272a4e0?

Reply:
89b272a looks much better, thanks!